src/util.js

```diff
@@ -430,6 +430,35 @@ var _ = $.extend(Mavo, {
 		return element.getAttributeNames().filter(name => regex.test(name));
 	},
 
+	// We need this to cache the results of the intense parsing operation in the following utility function
+	// E.g., { "svg": { "viewbox": "viewBox", ... }, "math": { ... } }
+	properlyCasedAttributesCache: {},
+
+	// Fixes the case of attributes that are not all-lowercase
+	// Especially useful for SVG attributes
+	// https://html.spec.whatwg.org/multipage/parsing.html#adjust-svg-attributes
+	getProperAttributeCase (element, attribute) {
+		const roots = "svg, math"; // Potential root elements
+
+		const root = element.closest(roots).tagName;
```

Comment:
This will return `null` if the element is not within `<svg>` or `<math>` and error. We could finish the selector with `:root` so that it always returns something, or use `?.` (depending on whether this is useful at all beyond SVG/MathML or not).

Reply:
I would follow the first route (with the `:root`), even though this utility is only used with SVG/MathML for now.

Thank you for pointing this out.